pyproject.toml

```diff
@@ -62,58 +62,27 @@ typish = '*'
 [tool.black]
 target-version = ['py38']
 
+[tool.pylint.'FORMAT']
+max-line-length = 120
+
 # FIXME: Pylint should not require this many exceptions
 [tool.pylint.'MESSAGES CONTROL']
 disable = '''
```

Comment:
This is the import section to look at. There are a couple that are ignored in code but these places are typically exceptions.